csharp/src/Drivers/Apache/Apache.Arrow.Adbc.Drivers.Apache.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="ApacheThrift" Version="0.21.0" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
This will need to be reverted and/or the branch rebased as I've already checked this in as a separate fix.

Reply:
Done.